ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-multi.yaml

```diff
@@ -0,0 +1,52 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-azure-417-multi
+promotion:
+  to:
+  - namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.17"
+  multi-latest:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.17"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- always_run: false
+  as: control-plane-multi-ipsec-3nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "3"
```

Comment:
Should be 1 or 2 if we want a 'multi' arch. And set ADDITIONAL_WORKERS to 2 or 1.

Reply:
And also the evns in https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/heterogeneous/ipi-install-heterogeneous-ref.yaml
```
  - name: ADDITIONAL_WORKERS
    default: "2"
    documentation: "Worker count for heterogeneous deploy"
  - name: ADDITIONAL_WORKER_ARCHITECTURE
    default: "aarch64"
    documentation: "Heterogeneous worker architecture, default set to arm64(aarch64)"
  - name: ADDITIONAL_WORKER_VM_TYPE
    default: "m6g.xlarge"
    documentation: "VM Instance type for Heterogeneous worker, default set to arm64 instance type"
```